tests/System/expected/test_webContext__TagManager.getAvailableVariableTypesInContext.xml

```diff
@@ -982,6 +982,22 @@
 						<introduction />
 						<condition />
 					</row>
+					<row>
+						<name>disableBrowserFeatureDetection</name>
+						<title>Disable Browser Feature Detection</title>
+						<value>0</value>
+						<defaultValue>0</defaultValue>
+						<type>boolean</type>
+						<uiControl>checkbox</uiControl>
+						<uiControlAttributes>
+						</uiControlAttributes>
+						<availableValues />
+						<description>Disable detection of visitor's current browser resolution and browser plugins supported.</description>
```

Comment:
We could say here: `By default, Matomo accesses some information from the visitor’s browser like the browser resolution. Some privacy regulations may only allow accessing information that is stored on a visitor’s device after having consent. If this applies to you, you can disable this feature.`.

The inline help we can remove.

Don't know if we can put links in here? I believe it maybe be possible in the inline help to use HTML.  If any possible be great to link to the FAQ: https://matomo.org/faq/how-to/how-do-i-disable-browser-feature-detection-completely/

Like `Learn more`...

Reply:
@tsteur I modified the text to `By default, Matomo accesses some information from the visitor’s browser like the browser resolution. Some privacy regulations may allow access of such information from the visitor’s device only after having a consent. If this applies to you, you can disable this feature.`

Yes we can put links using `inlineHelp`, added a link to FAQ.

Also moved the variable and and calling of method in Variable and Tag as suugested